WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -96,6 +106,37 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
         return if (errorImageRes == null) this else this.error(errorImageRes)
     }
 
+    private fun GlideRequest<Drawable>.attachRequestListener(
+        requestListener: RequestListener?
+    ): GlideRequest<Drawable> {
+        return if (requestListener == null)
+            this
+        else this.listener(object : com.bumptech.glide.request.RequestListener<Drawable> {
+            override fun onLoadFailed(
+                e: GlideException?,
+                model: Any?,
+                target: Target<Drawable>?,
+                isFirstResource: Boolean
+            ): Boolean {
+                requestListener.onLoadFailed(e)
+                return false
```

Comment:
Just double checking, returning `false` here means that even though we are handling the error (or success for onResourceReady) we still want Glide to handle it as well, is that correct?

Reply:
Yes, that's correct.